src/Components/BatchComponentsRequestModel.cs

```diff
@@ -5,11 +5,13 @@
 //-----------------------------------------------------------------------
 namespace Vasont.Inspire.Models.Components
 {
+    using System;
     using System.Collections.Generic;
 
     /// <summary>
     /// This class contains properties required for the batch component API.
     /// </summary>
+    [Obsolete("Use Local DTOs", true)]
```

Comment:
Should we be putting the name of the DTO that's replacing this model in the Obsolete message?